app/socket-apis/__tests__/upsert-why.js

```diff
@@ -7,8 +7,10 @@ import { MongoMemoryServer } from 'mongodb-memory-server'
 import { ObjectId } from 'mongodb'
 
 const USER1 = '6667d5a33da5d19ddc304a6b'
-const POINT1 = new ObjectId('6667d688b20d8e339ca50020')
-const POINT2 = new ObjectId('6667e4eea414d31b20dffb2f')
+const POINT1 = '6667d688b20d8e339ca50020'
+const POINT2 = '6667e4eea414d31b20dffb2f'
+const POINT3 = '6667d5a33da5d19ddc304a7c'
+const PARENTID = 'parent-id'
```

Comment:
please make PARENTID = "69098c842fd795ccce88da65" so that it is a valid ObjectId.